lib/common/MongoDbTransactionStore.ts

```diff
@@ -158,11 +158,25 @@ export default class MongoDbTransactionStore implements ITransactionStore {
   }
 
   /**
-   * Gets a list of transactions that are in the desired transaction time
-   * @param transactionTime The transaction time to query for
+   * Gets a list of transactions between the bounds of transaction time. The smaller value will be inclusive while the bigger be exclusive
+   * @param beginTransactionTime The first transaction time to begin querying for
+   * @param endTransactionTime The transaction time to stop querying for
    */
-  public async getTransactionsByTransactionTime (transactionTime: number): Promise<TransactionModel[]> {
-    const transactions = await this.transactionCollection!.find({ transactionTime: { $eq: Long.fromNumber(transactionTime) } }).toArray();
+  public async getTransactionsByTransactionTime (beginTransactionTime: number, endTransactionTime: number): Promise<TransactionModel[]> {
+    let transactions: TransactionModel[] = [];
+    if (beginTransactionTime === endTransactionTime) {
+      // if begin === end or end is not provided, query for 1 transaction time
+      transactions = await this.transactionCollection!.find({ transactionTime: { $eq: Long.fromNumber(beginTransactionTime) } }).toArray();
+    } else {
+      const start = Math.min(beginTransactionTime, endTransactionTime);
+      const end = Math.max(beginTransactionTime, endTransactionTime);
```

Comment:
I am not sure I get the purpose of this.

Reply:
In case begin is greater than end. I like your comment on the function name and parameter change though. It removes the need for this min max calculation and implies that end will always be lt and begin will always be gte.